src/components/schemas/statements.yaml

```diff
@@ -0,0 +1,140 @@
+statements_in:
+  type: object
+  properties:
+    file_id:
+      allOf:
+        - $ref: './fields.yaml#/id_string'
+      example: fidftadfdsdf
+  required:
+    - file_id
+
+statements_out:
+  type: object
+  additionalProperties: False
+  properties:
+    id:
+      allOf:
+        - $ref: './fields.yaml#/id_string'
+      example: stmtwownwrng
+    org_id:
+      $ref: './fields.yaml#/org_id'
+    created_at:
+      $ref: './fields.yaml#/created_at'
+    updated_at:
+      $ref: './fields.yaml#/updated_at'
+    file_name:
+      type: string
+      description: |
+        Specifies the name of file uploaded by the user.
+      example: amex.csv
+    file_size:
+      type: number
+      description: |
+        Specifies the size of file uploaded by the user.
+      example: 21KB
+    no_of_debit:
+      type: integer
+      description: |
+        Number of debit transactions in the document uploaded by the user.
```

Comment:
Can we use `statement` everywhere instead of using `document` pls ?

Reply:
Okay sir